Dockerfile

```diff
@@ -16,8 +16,10 @@ RUN add-apt-repository ppa:pinepain/libv8-5.2 -y && \
     apt-get update
 
 RUN apt-get install -y --force-yes \
-    php7.1-fpm php7.1-dev \
-    php7.1-mysql php7.1-xml php-curl php-intl php-pear php-mbstring php7.1-gd
+    php7.1-fpm php7.1-dev php7.1-mysql php7.1-xml \
+    php-curl php-intl php-pear php-mbstring php7.1-gd php7.1-pgsql
+
+RUN phpenmod pdo_mysql && phpenmod pdo_pgsql
```

Comment:
Hey, thanks for the PR but running phpenmod is not part of our build process as we prefer to configure our Docker containers with project-specific php.ini files. I'm happy to add the Postgres extension if you remove the phphenmod line.